Elements/src/Model.cs

```diff
@@ -92,9 +93,9 @@ public void AddElement(Element element, bool gatherSubElements = true)
             // when all internal types have been updated to not create elements
             // during UpdateRepresentation. This is now possible because
             // geometry operations are reactive to changes in their properties.
-            if (element is GeometricElement)
+            if (element is GeometricElement element1)
```

Comment:
This is a cleanup suggested by the linter to use C#'s new pattern matching capability.

Reply:
This is great to know!